src/Debug.cc

```diff
@@ -814,29 +805,27 @@ int dbg_handle_debug_input()
 
 	if ( ! step_or_next_pending || g_frame_stack.back() != last_frame )
 		{
-		string context =
-			get_context_description(stmt, g_frame_stack.back());
+		string context = get_context_description(stmt, g_frame_stack.back());
 		debug_msg("%s\n", context.c_str());
 		}
 
 	step_or_next_pending = false;
 
-	PrintLines(loc.filename, loc.first_line,
-			loc.last_line - loc.first_line + 1, true);
+	PrintLines(loc.filename, loc.first_line, loc.last_line - loc.first_line + 1, true);
 	g_debugger_state.last_loc = loc;
 
 	do
 		{
-		// readline returns a pointer to a buffer it allocates; it's
-		// freed at the bottom.
+			// readline returns a pointer to a buffer it allocates; it's
```

Comment:
This is definitely wrong. These comments should stay at the original indentation.

Reply:
I can reproduce this one external to zeek, and it's something to do with the `ifdef` that immediately follows. It indents it correctly (or well, leaves the indentation alone) if i remove those lines.

Reply:
seems tolerable if it's very specific to this situation

Reply:
I'm inclined to just leave it alone and file a bug on the clang-format side. It's pretty specific and it can just be fixed in a later version of clang-format.